fileonfeature.txt

```diff
@@ -1,2 +1,3 @@
 this is my file on feature branch
 added some lines to edit the file
+new line
```

Comment:
this is not the way ho we add new lines to our txt file
it should start with "added" for e.g.

Reply:
ok , i'm going to change it